template/README.md

```diff
@@ -2,19 +2,124 @@
 
 A brief description of My Project.
 
+## Using This Template
+
+Remove this section after initial setup!
+
+Search for and replace the following placeholders:
+
+* \<organization>
+* \<project>
+* \<project local url>
+* \<local environment>
+* \<project key>
+* \<branch>
+* \<provide link>
+
 ## Getting Started
 
-This project is based on BLT, an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices.
+This project is based on BLT, an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices. While this is one of many methodologies, it is our recommended methodology. 
+
+* Review the [Required / Recommended Skills](http://blt.readthedocs.io/en/latest/readme/skills) for working with a BLT project
+* Ensure that your computer meets the minimum installation requirements (and then install the required applications). See the [System Requirements](http://blt.readthedocs.io/en/latest/INSTALL/#system-requirements).
+* Request access to organization that owns the project repo in GitHub (if needed)
+* Fork the project repository in GitHub
+* Request access to the Acquia Cloud Environment for your project (if needed)
+* Setup a SSH key that can be used for GitHub and the Acquia Cloud (you CAN use the same key)
+    * [Setup GitHub SSH Keys](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
+    * [Setup Acquia Cloud SSH Keys](https://docs.acquia.com/acquia-cloud/ssh/generate)
+* Clone your fork
+```
+$ git clone git@github.com:<account>/<project>.git
+```
+* Add the parent repository as an upstream
+```
+$ git remote add upstream git@github.com:<organization>/<project>.git
+```
+* Install Composer Dependencies (warning: this can take some time based on internet speeds)
+```
+$ composer install
+```
+* Setup Local Environment 
+
+Note: Drupal VM is OPTIONAL but templated within BLT. If using Drupal VM, BLT 9 assumes all commands are run INSIDE the VM (this is a change from BLT 8).
+
+```
+$ blt vm
+```
+
+Warning: this command can take a long period of time (~15-20 minutes) the first time it is run, per project, depending on internet connectivity. 
+
+
+* Run the initial Setup
+```
+$ vagrant ssh
+$ blt setup
+``` 
+* Access the site and do necessary work at \<project local url> by running
+
+```
+$ drush @<project>.local uli
```

Comment:
We're trying to move away from using drush aliases for the VM. This also assumes that drush is globally installed.

Reply:
@grasmash will update these. although, you still need... @self to run in the vm, right?